lib/traject/macros/csv.rb

```diff
@@ -8,8 +8,22 @@ def column(header_or_index, options = {})
       lambda do |row, accumulator, _context|
         result = Array(row[header_or_index].to_s)
         result = result.map(&:strip) if options.key?(:trim)
+        result = (result.map { |s| s.split(options[:split]) }).first if options.key?(:split)
```

Comment:
Otherwise the accumulator value becomes [['value 1', 'Value 2']] instead of ['Value 1', 'Value 2'].